packages/testsuite/cypress/e2e/elytron-oidc-client/test-oidc-security-configure-oidc.cy.ts

```diff
@@ -0,0 +1,158 @@
+describe("TESTS: Configuring Elytron OIDC client in webconsole", () => {
+  const oidcAddress = ["subsystem", "elytron-oidc-client"];
+  const providerAddress = ["subsystem", "elytron-oidc-client", "provider", "keycloak"];
+  const secureDeploymentAddress = ["subsystem", "elytron-oidc-client", "secure-deployment", "wildfly-console"];
+  const secureServerAddress = ["subsystem", "elytron-oidc-client", "secure-server", "wildfly-management"];
+  const coreSeviceRbacAddress = ["core-service", "management", "access", "authorization"];
+  const mappedRoleAddress = [
+    "core-service",
+    "management",
+    "access",
+    "authorization",
+    "role-mapping",
+    "Administrator",
+    "include",
+    "user-userwithmappedrole",
+  ];
+  const optionalFields = "a.field-section-toggle-pf";
+  const activationRBACWarning = "#hal-finder-preview > div:nth-child(2) > a";
+  const activationRBACWizadrWarning = "#hal-modal > div > div > div.modal-body > p > p:nth-child(2)";
+  const activationRBACWarningMessage = "be sure your configuration has a user who will be mapped to one of the RBAC roles.";
+  const providerSelector = "#model-browser-root___provider_anchor";
+  const secureDeploymentSelector = "#model-browser-root___secure-deployment_anchor";
+  const secureServerSelector = "#model-browser-root___secure-server_anchor";
+
+  let wildfly: string;
+
+  const providerForm = {
+    id: "model-browser-root-provider-add",
+    providerName: "name",
+    providerValue: "keycloak",
+    serverUrl: "provider-url",
+    serverUrlValue: "http://localhost:8888/realms/wildfly-infra",
+  };
+
+  const secureDeployment = {
+    id: "model-browser-root-secure-deployment-add",
+    resource: "name",
+    provider: "provider",
+    clientId: "client-id",
+    publicClient: "public-client",
+  };
+
+  const secureServer = {
+    id: "model-browser-root-secure-server-add",
+    resource: "name",
+    provider: "provider",
+    clientId: "client-id",
+    bearerOnly: "bearer-only",
+    principal: "principal-attribute",
+    sslRequired: "ssl-required",
+  };
+
+  const roleMapping = {
+    id: "user-form",
+    userName: "name",
+    include: "include",
+  };
+
+  before(() => {
+    cy.startWildflyContainer().then((result) => {
+      wildfly = result as string;
+    });
+  });
+
+  after(() => {
+    cy.task("stop:containers");
+  });
+
+  beforeEach(() => {
+    // required for "flip" method to see all fields in form with a lots of elements
+    cy.viewport(1000, 2500);
+  });
+
+  it("Navigate to elytron OIDC client configuration from home page", () => {
+    cy.navigateTo(wildfly, "home");
+    cy.get("#tlc-configuration").should("be.visible").click();
+    cy.get("#subsystems").should("be.visible").click();
+    cy.get("#elytron-oidc-client").should("be.visible").click();
+    cy.get("#elytron-oidc-client > a:nth-child(4)").should("be.visible").click();
```

Comment:
> `a:nth-child(4)`

Why not just `a`? There is just one link.

Reply:
The html selector updated.